### PR TITLE
[STRATO-1790] Configuration of Selenium

### PR DESCRIPTION
**Pull request by Tanooj0902**

### CODE REVIEW

File: fixtures/framework/README.md [hunk 1]
```diff
@@ -215,3 +215,13 @@ After that, when the Application is already running, the certificate will be aut
 
 For steps to use letsencrypt tool please refer to <dir>/nginx-docker/letsencrypt/README.md
 
+#### Selenium tests
+
+Install chrome webdriver on your machine then follow these steps:
+
+```
+cd <dir>/<dir>-server
```
Comment (nikitamendelbaum) [inline fixtures/framework/README.md:223]: @Tanooj0902 I am confused - shouldn't the selenium tests be in the UI part of the app instead? The server (backend) part is only exposing the APIs - why do we need selenium for that?

Comment (Tanooj0902) [reply fixtures/framework/README.md:223]: There are a few things that lead me to move the selenium to the sever. 

1. While configuring the selenium I refer to the ECHO. Their hierarchy is top-level package.json. then UI package.json is different. there is no server package.json. we all are doing with the root package.json.

2. If we move selenium to the UI. We need packages mocha, selenium-webdriver and if we want ES6 syntax then we need to eject the babel in react application. Here, I am avoiding mocha, mocha-babel, and selenium-webdriver to be installed on UI as well. 

3. Also, While writing selenium test cases in the selenium we definitely need the config.yaml in the future. So, I thought to move to the server folder. 

Let me know your thoughts. I am still open to discussion.

Comment (nikitamendelbaum) [reply fixtures/framework/README.md:222]: Hmm. I am confused why selenium scripts need config.yaml
My understanding of selenium is that it simulates the user behavior in the browser interacting with the app.
We don't want selenium to interact with backend. Just think of it as the "robot user in the browser".

ECHO had the server's package.json in root directory but it always seemed wrong to me.

I don't want to make the server part even more complicated for the external developers.
If it takes too much to add Selenium to UI part - let's better have it as a separate service along with backend and ui. So feel free to add the `selenium` directory in the root of the app fixtures.

Comment (Tanooj0902) [reply fixtures/framework/README.md:222]: Sounds good.

**Pull request merged**